logging/config/cluster-logging-curator.adoc

```diff
@@ -5,14 +5,16 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} {product-version}, log curation is performed by Elasticsearch Curator, based on a xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retention policy] that you define.
+In {product-title} {product-version}, log curation is performed by ElasticSearch `IndexManagement` - the `elasticsearch-im-app`, `elasticsearch-im-audit` and `elasticsearch-im-infra` cronjobs, which you can configure using the xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retentionPolicy].
 
-The Elasticsearch Curator tool removes Elasticsearch indices that use the data model prior to {product-title} 4.5. You can modify the Curator index retention policy for your old data.
+You can configure the `IndexManagement` jobs to delete Elasticsearch data that uses both the current data model and the data model prior to {product-title} 4.5.
+// Preserve the hard-coded "4.5" in the preceding paragraph. Do not replace it with a product version attribute.
 
-// The following include statements pull in the module files that comprise
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
+Optionally, instead of `IndexManagement`, you can use the Elasticsearch Curator to perform log curation. The Elasticsearch Curator will be deprecated in a {product-title} version 4.7 (OpenShift Logging 5.0) and removed in OpenShift Logging 5.1.
```

Comment:
Doesnt this imply you can use either to accomplish the same task?
* Curator (as we use it) will only remove indices in the old data model format
* IM will remove indices regardless of the the data model

Essentially, we should discourage users from curator use for 4.5 and later

Reply:
Thanks, Jeff. I've clarified this in the updated files.